src/js/pagination/specs/paging-collection-spec.js

```diff
@@ -89,6 +96,28 @@ define(['jquery',
                 }
             ));
 
+            it('doesn\'t send unnecessary parameters in request (total pages, total records)', AjaxHelpers.requests(
```

Comment:
Why is the actual query parameter ``total_entries`` but the title says ``total records``?